docs/docsets/.docset/Contents/Resources/Documents/undocumented.json

```diff
@@ -1,5 +1,12 @@
 {
   "warnings": [
+    {
```

Comment:
We should probably create a backlog ticket to go through all these and add appropriate documentation